src/extractors/amazon.js

```diff
@@ -123,15 +122,10 @@ function getAudibleDetails() {
       return;
     }
 
-    // logMarian(label, includedLabels.includes(label));
-    // Print debug info for labels not included
-    // skip if not included in the list
-    if (!includedLabels.includes(label)) {
-      // logMarian(`Label not currently included: "${label}"`);
-      return;
-    }
-
-    if (label === 'Audible.com Release Date') {
+    // Match any Audible.<TLD> Release Date
+    if (/^Audible\.[a-z]{2,3} Release Date$/i.test(label)) {
```

Comment:
This doesn't take into account for domains like audible.co.uk or audible.co.jp or the weirder audible.com.au

Reply:
I think a better regex would be `/^Audible\.[^\s]+ Release Date$/i`